CMakeLists.txt

```diff
@@ -46,6 +48,15 @@ set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
 add_subdirectory(libraries)
 add_subdirectory(plugins)
 
+# Create and install CMake configuration files for this project that are
+# necessary for other projects to call find_package(gz-sim-yarp-plugins).
+include(InstallBasicPackageFiles)
+Install_basic_package_files(${PROJECT_NAME}
+                            VERSION ${${PROJECT_NAME}_VERSION}
+                            COMPATIBILITY AnyNewerVersion
+                            VARS_PREFIX ${PROJECT_NAME}
+                            NO_CHECK_REQUIRED_COMPONENTS_MACRO)
```

Comment:
Should I add the `DEPENDENCIES` argument here? http://robotology.github.io/ycm/gh-pages/latest/module/InstallBasicPackageFiles.html

Reply:
Yes, you should manually add any library that we are using in public headers you install, i.e. looking at https://github.com/robotology/gz-sim-yarp-plugins/blob/main/libraries/device-registry/DeviceRegistry.hh#L1 both `YARP` and `gz-sim8` . However, probably we can remove the `gz/common/Event.hh` include in https://github.com/robotology/gz-sim-yarp-plugins/blob/main/libraries/device-registry/DeviceRegistry.hh#L1 that is not used, so we can just add `YARP` to `DEPENDENCIES`.

Reply:
Perfect thanks!

Reply:
Solved in https://github.com/robotology/gz-sim-yarp-plugins/pull/160/commits/4bff351a228767ce8324797b91d27476f432502f and https://github.com/robotology/gz-sim-yarp-plugins/pull/160/commits/78cd901b6d5a6f7ad45785f0276426701cab2f1b